app/src/main/kotlin/at/bitfire/davdroid/ui/intro/BatteryOptimizationsPage.kt

```diff
@@ -133,7 +101,7 @@ class BatteryOptimizationsPage: IntroPage {
     @HiltViewModel
     class Model @Inject constructor(
```

Comment:
By the guideline: Rename the Model to `BatteryOptimizationsPageModel` and put it in a file of the same name.